include/orbis/_types/rtc.h

```diff
@@ -0,0 +1,3 @@
+typedef struct OrbisRtcTick {
+	uint64_t Ticks;
+} SceRtcTick;
```

Comment:
I think this should be:

```suggestion
}OrbisRtcTick;
```

Reply:
Right, missed that, fixing now.